src/ssl.c

```diff
@@ -7899,6 +7922,7 @@ int wolfSSL_DTLS_SetCookieSecret(WOLFSSL* ssl,
             }
 #endif /* NO_HANDSHAKE_DONE_CB */
 
+#ifndef WOLFSSL_WPAS
```

Comment:
This is not a runtime check.  Why would we assume that the WOLFSSL_WPAS build time flag guarantees WPAS mode at runtime?

What about a user doing TLS with this build but not as part of a WPAS connection.